src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -46,6 +46,43 @@ def dialog_content
       )
     end
 
+    def help_handler
+      Yast::Popup.LongText(
+        _("Help"),
+        RichText(help_text),
+        60,
+        20
+      )
+    end
+
+    def help_text
+      _("<p>The alternatives system is used when some equivalent applications are " \
+        "installed in the system. For example if several implementations and versions of Java " \
+        "are installed, the alternatives system will be used to decide which one to use when " \
+        "the command \"java\" is called.</p><p>Every alternative have an " \
+        "status, it can be automatic or manual. In <b>automatic mode</b> the system uses " \
+        "the choice with higher priority. Note that <b>priority</b> is set by the packages " \
+        "creators trying to provide reasonable defaults. The <b>manual mode</b> is set " \
+        "when the user selects a choice manually.</p><p>To change the current choice of an " \
+        "alternative, select the alternative, click the <b>edit</b> button then select the " \
+        "desired choice in the table and click on the <b>set choice</b> button. If you want to " \
+        "have the alternative in automatic mode just click the <b>set automatic mode</b> button " \
+        "in the edit dialog.</p><p>Furthermore every choice has its own set of <b>slaves</b>, " \
+        "these will be applied in the system when the choice is selected.</p><p>Lets see an " \
```

Comment:
It would be "let's see" (missing apostrophe). But even though, it looks quite informal compared to the rest of the text.

What about something more simple and formal? "For example, given an alternative..."